common/pb/session_commands.proto

```diff
@@ -164,6 +166,8 @@ message Command_AccountPassword {
     }
     optional string old_password = 1;
     optional string new_password = 2;
+    // optional string hashed_old_password = 3; // we don't want users to steal hashed passwords and change them
```

Comment:
This wasn't apart of the protocol yet, was it? Why do we comment out and remove 3 from service?

Reply:
in case we'd ever want it, I already implemented it at that point and the comment makes clear we don't want it right now.

Reply:
I'm not sure of a situation where we'll ever use it, and the comment isn't explicit on why it's reserved